src/shared-components/callout/style.js

```diff
@@ -0,0 +1,34 @@
+import styled from '@emotion/styled';
+
+import { COLORS, SPACER, TYPOGRAPHY_CONSTANTS } from '../../constants';
+
+export const ParentContainer = styled.div`
+  max-width: 327px;
+`;
+
+export const CalloutContainer = styled.div`
+  background-color: ${COLORS.infoBackground};
+  padding: ${SPACER.medium};
+  border-radius: 8px;
```

Comment:
here it sits at `8px`

Reply:
Oh oops, sorry. It just looked big. I'm gonna approve